### ISSUE TITLE
Menu component does not honor a11yTitle text

### ISSUE THREAD

**Issue by LiveFreeAndRoam** [grommet/grommet#3525]
Attempts to set the a11yTitle on a menu component does not work.  Instead, the aria-label appears to be hardcoded to "Open Menu"
 
### Expected Behavior
I expected the HTML button's aria-label to be set the same as my Grommet's Menu a11yTitle text.

### URL, screen shot, or Codepen exhibiting the issue

https://codesandbox.io/s/menu-a11ytitle-not-honored-48mq7

Use developer tools to observe the HTML and aria-label.

### Steps to Reproduce

1. 2. 3.

### Your Environment

- Grommet version: v2
- Browser Name and version: Chrome
- Operating System and version (desktop or mobile): Windows 10.

**Comment by taysea** [comment]
Hey there, thanks for bringing this up. At the current time, the way to adjust the aria-label would be to use the `messages` prop (https://v2.grommet.io/menu#messages). By default the control button has 'Open Menu' so that users with screen readers can receive an indication of what pressing the button will do, but you are welcome to change it to the label on your menu or anything you'd like. Hopefully that allows you to make a fix for now. I will look into why we also accept an a11yTitle on this component even though it has no effect. Thanks!

**Comment by taysea** [comment]
@ShimiSun I think we can close this now that my PR merged : )
